tests/suites/test_suite_pk.function

```diff
@@ -1398,7 +1398,7 @@ exit:
 /* END_CASE */
 
 /* BEGIN_CASE depends_on:MBEDTLS_PK_PARSE_C:PSA_WANT_ALG_SHA_256:MBEDTLS_TEST_PK_PSA_SIGN */
-void pk_psa_sign(int psa_type, int bits, int rsa_padding)
+void pk_psa_sign(int psa_type, int bits)
```

Comment:
Well this is the one case (`pk_wrap_psa()`) where an RSA key can have PSS as its primary algorithm, so I think we should keep PSS testing here. (Probably need to adjust the body of the function though.)

Btw, pre-existing: the comments still say "opaque" to mean a key that was created with `wrap_psa()` (previously called `setup_opaque()`). Perhaps we should change that to "PSA-wrapping"?